src/main/kotlin/it/pagopa/ecommerce/helpdesk/utils/v2/ResponseBuilderV1.kt

```diff
@@ -33,6 +38,66 @@ fun buildTransactionSearchResponse(
         )
         .transactions(results)
 
+// TODO
```

Comment:
forgotten comment or are we missing something?

Reply:
we inserted this TODO for a second check and we forgot to remove it.

Reply:
Resloved [here](https://github.com/pagopa/pagopa-ecommerce-helpdesk-service/pull/124/commits/6545645d7431833e4a8add069ceb31c48f50be93)